### DISCUSSION TITLE
Chrome 插件的OLLAMA选项？

### DISCUSSION THREAD

**Discussion by lesca** [nextai-translator/nextai-translator#1313] [Q&A]
你好，

请问使用chrome插件时，能否使用本地模型？我下载安卓的WINDOWS桌面客户端里有这个选项，但是扩展插件里又没有了。？

![image](https://github.com/openai-translator/openai-translator/assets/5976219/83097183-9d39-471f-afcb-a99137c47aec)

**Comment by yetone** [comment]
因为 Ollama 跨域很麻烦，所以浏览器插件暂不支持 Ollama

**Comment by lesca** [comment]
> 因为 Ollama 跨域很麻烦，所以浏览器插件暂不支持 Ollama

不明觉厉。加油！希望以后能支持！

**Comment by lesca** [comment]
@yetone ollama 可以设置 `OLLAMA_ORIGINS` 环境变量 

Windows上设置
```
set OLLAMA_ORIGINS=*
```

浏览器插件就可以访问OLLAMA，希望能够支持。

参考
[1] [How can I allow additional web origins to access Ollama](https://github.com/ollama/ollama/blob/main/docs/faq.md#how-can-i-allow-additional-web-origins-to-access-ollama)

**Comment by yetone** [comment]
@lesca 哪怕多了这么一步，用户就不会设置了，如果多了这么一步用户也会设置为什么用户不直接使用 ollama 的 OpenAI compatible 的 API 进行设置呢，所以感觉是有点悖论在这里面的

**Reply by lesca** [reply]
正解，现在就是用 ollama 的 OpenAI compatible 的 API 作为过渡。方法给大家参考一下
